components/micro_wake_word.rst

```diff
@@ -0,0 +1,106 @@
+Micro Wake Word
+================
+
+.. seo::
+    :description: Instructions for creating a custom wake word using microWakeWord.
+    :image: voice-assistant.svg
+
+ESPHome implements an on-device wake word detection framework from `microWakeWord <https://github.com/kahrendt/microWakeWord>`__.
+This repository/library allows you to create a custom wake word for your ESPHome device.
+
+The training process is described on the `microWakeWord GitHub repository <https://github.com/kahrendt/microWakeWord>`__.
+
+The ``micro_wake_word`` component requires an **ESP32-S3** to function.
+
+.. code-block:: yaml
+
+    micro_wake_word:
+      model: hey_jarvis
+
+    micro_wake_word:
+      model: github://esphome/micro-wake-word-models/models/hey_jarvis.json
```

Comment:
Can we default this to Ok, Nabu

Reply:
It's also not clear here that these are 2 examples and that either can work.

Reply:
```suggestion
    micro_wake_word:
      model: okay_nabu

    micro_wake_word:
      model: github://esphome/micro-wake-word-models/models/okay_nabu.json
```

Reply:
Yes, can do. I will also need to update the trained model in the repo to the latest version